test/integration/scheduler_perf/scheduler_bench_test.go

```diff
@@ -45,41 +49,87 @@ func BenchmarkScheduling(b *testing.B) {
 	for _, test := range tests {
 		name := fmt.Sprintf("%vNodes/%vPods", test.nodes, test.existingPods)
 		b.Run(name, func(b *testing.B) {
-			benchmarkScheduling(test.nodes, test.existingPods, test.minPods, setupStrategy, testStrategy, b)
+			benchmarkScheduling(test.nodes, test.existingPods, test.minPods, defaultNodeStrategy, setupStrategy, testStrategy, b)
 		})
 	}
 }
 
-// BenchmarkSchedulingAntiAffinity benchmarks the scheduling rate of pods with
+// BenchmarkSchedulingPodAntiAffinity benchmarks the scheduling rate of pods with
 // PodAntiAffinity rules when the cluster has various quantities of nodes and
 // scheduled pods.
-func BenchmarkSchedulingAntiAffinity(b *testing.B) {
+func BenchmarkSchedulingPodAntiAffinity(b *testing.B) {
 	tests := []struct{ nodes, existingPods, minPods int }{
 		{nodes: 500, existingPods: 250, minPods: 250},
 		{nodes: 500, existingPods: 5000, minPods: 250},
```

Comment:
Now that you are at it, could you add nodes: 1000, existing: 1000, minPods: 500 test here and in the next test?

Reply:
@bsalamat to confirm, would you like the following tests to be added this 1000-node test entry?

- BenchmarkSchedulingPodAntiAffinity
- BenchmarkSchedulingPodAffinity
- BenchmarkSchedulingNodeAffinity

And after that, our cases would be like this:

- BenchmarkScheduling (have no affinity)
    
    ```go
    {nodes: 100, existingPods: 0, minPods: 100},
    {nodes: 100, existingPods: 1000, minPods: 100},
    {nodes: 1000, existingPods: 0, minPods: 100},
    {nodes: 1000, existingPods: 1000, minPods: 100},
    ```

- BenchmarkSchedulingPodAntiAffinity

    ```go
    {nodes: 500, existingPods: 250, minPods: 250},
    {nodes: 500, existingPods: 5000, minPods: 250},
    {nodes: 1000, existingPods: 1000, minPods: 500},
    ```

- BenchmarkSchedulingPodAffinity

    ```go
    {nodes: 500, existingPods: 250, minPods: 250},
    {nodes: 500, existingPods: 5000, minPods: 250},
    {nodes: 1000, existingPods: 1000, minPods: 500},
    ```

- BenchmarkSchedulingNodeAffinity

    ```go
    {nodes: 500, existingPods: 250, minPods: 250},
    {nodes: 500, existingPods: 5000, minPods: 250},
    {nodes: 1000, existingPods: 1000, minPods: 500},
    ```

Reply:
Seems reasonable from my side :-)

Reply:
@Huang-Wei Looks good for now. I want to be able to try a larger number of nodes, but it has the risk of timing out in our CI/CD. Let's go with 1000 for now.

Reply:
Done. Thanks @resouer @bsalamat .